.scripts/run_osx_build.sh

```diff
@@ -11,7 +11,7 @@ MINIFORGE_HOME=${MINIFORGE_HOME:-${HOME}/miniforge3}
 ( startgroup "Installing a fresh version of Miniforge" ) 2> /dev/null
 
 MINIFORGE_URL="https://github.com/conda-forge/miniforge/releases/latest/download"
-MINIFORGE_FILE="Mambaforge-MacOSX-$(uname -m).sh"
+MINIFORGE_FILE="Miniforge3-MacOSX-$(uname -m).sh"
```

Comment:
Yeah that seems to have fixed the failures.

Reply:
They are deprecating Mambaforge since now it is identical to Miniforge3. This is a brownout period, and hence the failure

https://github.com/conda-forge/miniforge#download